fixtures/vcr_cassettes/panel_find.yml

```diff
@@ -0,0 +1,65 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://co1.qualtrics.com/API/v3/mailinglists/ML_00c5BS2WNUCWQIt
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      X-API-TOKEN:
+      - 6Wpo0Vsx1cN1kcHivCaGTz5IhOvchLrg1o4L0KOZ
+      User-Agent:
+      - Faraday v0.9.2
+  response:
+    status:
+      code: 200
+      message: 
+    headers:
+      content-type:
+      - application/json; charset=utf-8
+      request-time:
+      - '216'
+      content-length:
+      - '164'
+      date:
+      - Wed, 13 Apr 2016 13:15:28 GMT
+      connection:
+      - close
+    body:
+      encoding: UTF-8
+      string: '{"result":{"id":"ML_00c5BS2WNUCWQIt","libraryId":"UR_5dURLpfp5tm43EV","name":"Panel name","category":"Unassigned"},"meta":{"httpStatus":"200 -
+        OK"}}'
+    http_version: 
+  recorded_at: Wed, 13 Apr 2016 13:15:28 GMT
+- request:
+    method: get
+    uri: https://co1.qualtrics.com/API/v3/mailinglists/ML_00c5BS2WNUCWQI0
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      X-API-TOKEN:
+      - 6Wpo0Vsx1cN1kcHivCaGTz5IhOvchLrg1o4L0KOZ
+      User-Agent:
+      - Faraday v0.9.2
+  response:
+    status:
+      code: 400
+      message: 
+    headers:
+      content-type:
+      - application/json
+      content-length:
+      - '145'
+      date:
+      - Wed, 13 Apr 2016 13:15:35 GMT
+      connection:
+      - close
+    body:
+      encoding: UTF-8
+      string: '{"meta":{"httpStatus":"400 - Bad Request","error":{"errorMessage":"Invalid
+        mailingListId parameter. Mailing list not found","errorCode":"GP_0"}}}'
```

Comment:
That's quite weird. Qualtrics API should return 404 here to be consistent with other endpoints.